functions.py

```diff
@@ -0,0 +1,84 @@
+from math import remainder
```

Comment:
Don't do this. Function should not just float around in a file full of functions. Create logical entities that contain the functions and each has a single responsibility.